public/docs/_examples/i18n/ts/app/messages.fr.1.ts

```diff
@@ -0,0 +1,5 @@
+// #docregion
+export const TRANSLATION = `
+
```

Comment:
A file we need for example purpose, since our docs examples come from real files. Later in the guide we insert the content of the the `.xlf` file.